### DISCUSSION TITLE
Raster Analysis

### DISCUSSION THREAD

**Discussion by madixfun3** [TerriaJS/terriajs#5372] [General]
Hi,
I am new to Terria but I am curious if someone can guide me to some information about how to code Terria to perform raster analysis. 

I am cannot wait to be part of this community !

**Comment by DavidHine** [comment]
Hi Madixfun
Terriajs seems primarily a spatial data cataloguing (providing easy access like a library) and spatial data visualisation application. It hoes have some datum transformation tools. In short it doesn't offer anything that I know of to assist in raster analysis. 

I think you will need to do the raster analysis in another application. You may already have ideas on analysis but you could try QGIS, SAGA or even R (the statistical language) though R has a learning curve that is steeper than the others.

regards
David Hine
